### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by nwlandry**
I found a few minor things that I wanted to fix/improve.

### CODE REVIEW

File: docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst [hunk 1]
```diff
@@ -46,3 +46,4 @@
 
       ~Hypergraph.copy
       ~Hypergraph.dual
+      ~Hypergraph.cleanup
```
Comment (leotrs) [inline docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst:49]: Is this typo? This shouldn't be here, let's leave it under "methods that modify the structure".

**Review by leotrs — APPROVED** [review]

File: docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst (hunk 1 above)
Comment (nwlandry) [reply docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst:49]: Depending on the `in_place` argument, a hypergraph may be returned. Happy to omit, however.

Comment (leotrs) [reply docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst:49]: Oh snap, you're right. Uhhh let's just omit it for now.

Comment (nwlandry) [reply docs/source/api/classes/xgi.classes.hypergraph.Hypergraph.rst:49]: Okay!

**Review by leotrs — APPROVED** [review]

**Pull request merged**